dashboards/google-vertex-ai/vertex-ai-provisioned-throughput-utilization.json

```diff
@@ -0,0 +1,112 @@
+{
+  "displayName": "Provisioned Throughput Utilization",
+  "dashboardFilters": [],
+  "labels": {},
+  "mosaicLayout": {
+    "columns": 48,
+    "tiles": [
+      {
+        "height": 26,
+        "width": 48,
+        "widget": {
+          "title": "",
+          "id": "",
+          "xyChart": {
+            "chartOptions": {
+              "displayHorizontal": false,
+              "mode": "COLOR",
+              "showLegend": false
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "legendTemplate": "${resource.labels.model_user_id} ${resource.labels.location}${resource.labels.monitored_resource_container}",
+                "measures": [],
+                "minAlignmentPeriod": "60s",
+                "plotType": "LINE",
+                "targetAxis": "Y2",
+                "timeSeriesQuery": {
+                  "outputFullDuration": false,
+                  "timeSeriesFilterRatio": {
+                    "numerator": {
+                      "aggregation": {
+                        "alignmentPeriod": "60s",
+                        "crossSeriesReducer": "REDUCE_MAX",
+                        "groupByFields": [
```

Comment:
this groupBy (and each case of it below) omits a couple labels that could theoretically be part of the unique identifier for a PublisherModel, for example `model_version_id` and `publisher`. Should these be added in for completeness?